# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/WarpCacheIRTranspiler.cpp
# Commit: cf96078694ae
# Full Hash: cf96078694aea74cae4557f683a76c6de6ba9cf1
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2023-08-05 09:19:01
# Regressor Bug: 1692065
# File Overlap Count: 3
# Description:
#   Bug 1692065 - wasm: Rename MIRType::RefOrNull to 'WasmAnyRef'. r=yury
#   
#   This is a mechanical change. The previous name 'RefOrNull' was too
#   generic, and it wasn't clear that the type was actually a wasm::AnyRef
#   type.
# ==============================================================================

diff -r 662cb63c42d7 -r cf96078694ae js/src/jit/WarpCacheIRTranspiler.cpp
--- a/js/src/jit/WarpCacheIRTranspiler.cpp	Fri Aug 04 14:06:49 2023 +0000
+++ b/js/src/jit/WarpCacheIRTranspiler.cpp	Fri Aug 04 14:06:49 2023 +0000
@@ -5390,7 +5390,7 @@
       MOZ_CRASH("Unexpected type for Wasm JitEntry");
     case wasm::ValType::Ref:
       // Transform the JS representation into an AnyRef representation.
-      // The resulting type is MIRType::RefOrNull.  These cases are all
+      // The resulting type is MIRType::WasmAnyRef.  These cases are all
       // effect-free.
       switch (arg->type()) {
         case MIRType::Object: